docs/semgrep-ci/semgrep-ci-with-app.md

```diff
@@ -0,0 +1,332 @@
+---
+slug: semgrep-ci-with-app
+append_help_link: true
+description: ""
+title: Running Semgrep in CI with Semgrep App
+hide_title: true
+---
+
+import MoreHelp from "/src/components/MoreHelp"
+import CiScheduling from "/src/components/CiScheduling.mdx"
+import CiIgnoringFiles from "/src/components/CiIgnoringFiles.mdx"
+import DiffAwareScanning from "/src/components/DiffAwareScanning.mdx"
+
+# Running Semgrep in continuous integration (CI) with Semgrep App
+
+Run Semgrep in your continous integration (CI) pipeline to scan your repository for code vulnerabilities and other issues. Connect your CI pipeline with Semgrep App to:
+
+* Block pull or merge requests (PRs or MRs) based on the rule that generated the finding.
+* Scan many repositories and manage their findings in bulk.
+* Triage (ignore) false-positive findings from low-performing rules.
+* Fork existing rules to create custom rules and add them to Semgrep App for scanning.
+
+This guide explains how to connect your repository to Semgrep App to scan continuously.
+
+:::note
+* This guide's configuration and feature support are specific to Semgrep-App-connected CI jobs. Refer to [Running Semgrep in CI without Semgrep App](../manual-ci-setup) for stand-alone CI jobs.
+* Semgrep 0.98.0 introduced changes to how certain CI providers fetch environment variables. Refer to the appendix at the end of this document for more information.
+:::
+
+The following video walks you through setting Semgrep in your CI through Semgrep App.
+
+<iframe class="yt_embed" width="100%" height="432px" src="https://www.youtube.com/embed/ukIUM3j0gZY" frameborder="0" allowfullscreen></iframe>
+
+
+## Feature support
+
+Support for certain features of Semgrep App depend on your CI provider or source code management tool (SCM). The following table breaks down the features and their availability:
+
+| Feature | GitHub | GitLab | BitBucket | CI Provider support |
+| ------- | -------- | ------- | -------- | ---------------- |
+| **Diff-aware scanning** | ✅ Yes | ✅ Yes | ✅ Yes  | ✅ Available (may need additional set up) | 
+| **Hyperlinks** | ✅ Yes | ✅ Yes | ✅ Yes  |  ✅ Available (may need additional set up) |
+| **SCM security dashboard** |  ✅ GitHub Advanced Security Dashboard |  ✅ GitLab SAST Dashboard | ❌ No | ❗ Only GitHub Actions and GitLab CI/CD |
```

Comment:
In GitLab I believe the dashboard is called the [GitLab Security Dashboard](https://docs.gitlab.com/ee/user/application_security/security_dashboard/index.html).

Reply:
Added in latest. Resolving.